Products/SQLAlchemyDA/da.py

```diff
@@ -65,12 +103,13 @@ class SAWrapper(SimpleItem, PropertyManager):
     quoting_style = 'standard'
     _isAnSQLConnection = True
     extra_engine_options = ()
-
+    
     security = ClassSecurityInfo()
 
     def __init__(self, id, title=''):
         self.id = id
         self.title = title
+        register_da(self.id, self)
```

Comment:
I hadn't intended to allow that, but it's a good point...I should document the behavior that any subsequent registration under the same name will over-write the previous registration. If anyone else in the world ever uses this API and wants a different behavior, I'd be glad to consider other options.

Another question about this line of code is whether it gets called when unpickling after shutting down and starting up. I don't think my tests cover that. I need to work on that, though I'm not entirely clear on how to set up that test.